app/jobs/hyku_addons/reindex_available_works_job.rb

```diff
@@ -21,7 +29,8 @@ def reindex(cname)
           next unless klass.count.positive?
 
           Rails.logger.debug "==== Queue ReindexModelJob for #{klass} ==="
-          HykuAddons::ReindexModelJob.perform_later(model_class, cname)
+          HykuAddons::ReindexModelJob.perform_later(model_class, cname, options: { cname_doi_mint: @cname_doi_mint }) if @cname_doi_mint.include?(cname)
```

Comment:
I know that Hyrax/Hyku do this a lot, add extra conditions at the end of lines for `if/else`, but it woudl be clearer if we instead laid out a concise condition: 

```
options = @cname_doi_mint.include?(cname) ? { options: { cname_doi_mint: @cname_doi_mint } } : {}

HykuAddons::ReindexModelJob.perform_later(model_class, cname, options) 
```

Reply:
I think this will do:
               HykuAddons::ReindexModelJob.perform_later(model_class, cname, options: { cname_doi_mint: @cname_doi_mint })

@cname_doi_mint is always an empty array, so I am not sure we need the check here:
    options = @cname_doi_mint.include?(cname) ? { options: { cname_doi_mint: @cname_doi_mint } } : {}